site-src/guides/index.md

```diff
@@ -313,7 +313,7 @@ A cluster with:
    The following instructions assume you would like to cleanup ALL resources that were created in this quickstart guide.
    Please be careful not to delete resources you'd like to keep.
 
-   1. Uninstall the InferencePool, InferenceModel, and model server resources
+   1. Uninstall the InferencePool, InferenceObjective and model server resources
```

Comment:
cleanup should use helm to uninstall inferencepool (mirror the install). 
additionally I noticed we delete all resources of all gateways. this doesn’t scale. 
maybe we should move the clean of each gateway to next step where each gateway cleanup section is specified